hack/tests/resources/kubemark-machine-deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
         kind: KubemarkMachineTemplate
         name: km-wl-kubemark-md-0
-      version: 1.29.8
+      version: 1.31.2
```

Comment:
I realize this was made before it was released, but 1.31.9's out. Do you want to use that one, or stick with this?

Reply:
i was having trouble getting more recent images for kind/kindest, so that's why i backed off to 1.31.2